CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 
 #### Minor Changes
 - Add new MDS fields and migrate existing MDS asset keys to mobilityDCAT-AP
+- Introduce a stable ID for contracts offers
```

Comment:
I believe we should move this to patch and prefix it for better classification by the reader. Our changelogs for this repo are from the "sovity EDC Community Edition" product perspective, so all technical changes need to be marked as such for normal readers.

From a product perspective, I'd also consider it a patch change:

My suggestion: `- DspCatalogService: Contract Offer IDs are now stable`